### PR TITLE
Converter for Pandora network data

### PR DESCRIPTION
**Pull request by mer-a-o**
## Description

Pandora data can be downloaded from https://data.pandonia-global-network.org/ using wget (`wget -np -r -nH --cut-dirs=3 --accept '*L2_rnvs3*txt' --reject 'index*' https://data.pandonia-global-network.org/` this will download all stations). 
This converter is designed to convert total column NO2 (`L2_rnvs3`) to the ioda format. The ioda file can then be used by `ColumnRetrieval` obs operator in UFO. 

## Issue(s) addressed

Resolves #<issue_number>

## Dependencies

List the other PRs that this PR is dependent on:
- [ ] ...

## Impact

Expected impact on downstream repositories:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have run the unit tests before creating the PR

### CODE REVIEW

File: src/compo/pandora_2ioda.py
Comment (jeromebarre) [inline src/compo/pandora_2ioda.py]: optional but:
```
import scipy.constants as sc

avogadro_constant = sc.N_A
```

File: src/compo/pandora_2ioda.py
Comment (jeromebarre) [inline src/compo/pandora_2ioda.py]: is there a way to reduce this line?

**Review by jeromebarre — APPROVED** [review]

File: src/compo/pandora_2ioda.py
Comment (BenjaminRuston) [inline src/compo/pandora_2ioda.py]:
```suggestion
            sections = content.split('-' * 87)
```

Comment (BenjaminRuston) [reply src/compo/pandora_2ioda.py]: @mer-a-o this should work

Comment (mer-a-o) [reply src/compo/pandora_2ioda.py]: Thanks @BenjaminRuston !

File: src/compo/pandora_2ioda.py
Comment (mer-a-o) [reply src/compo/pandora_2ioda.py]: Obs is already in `mol/m2`, I removed this line.

File: test/CMakeLists.txt
Comment (BenjaminRuston) [inline test/CMakeLists.txt]:
```suggestion
                          --time_range 2023080115 2023080121"
```

Personal preference, but I usually try to minimize the use of the shorthand options as they can be confusing, but feel `-i` and `-o` are always good to leave shorthand

File: src/compo/pandora_2ioda.py
Comment (BenjaminRuston) [inline src/compo/pandora_2ioda.py:262]:
```suggestion
    required.add_argument(
        '--time_range',
```

this actually looks like a required argument

File: src/compo/pandora_2ioda.py
Comment (BenjaminRuston) [inline src/compo/pandora_2ioda.py]:
```suggestion
        "format --time_range YYYYMMDDHH YYYYMMDDHH",
```

File: src/compo/pandora_2ioda.py
Comment (BenjaminRuston) [inline src/compo/pandora_2ioda.py:266]: I see this will have it get the whole file if no argument is given... sorry,,, reading as I go.

File: src/compo/pandora_2ioda.py
Comment (BenjaminRuston) [inline src/compo/pandora_2ioda.py]:
```suggestion
        '--time_range',
```

**Review by BenjaminRuston — APPROVED** [review]
run the converter using`--time_range` and it worked just fine and produced the output in the `testoutput` file

**Comment by BenjaminRuston** [comment]
@mer-a-o only final suggestion which is your call is to replace `-r` with `--time_range` ,, and as I"m thinking on the fly maybe `--date_range` and not have the shorthand `-r`

**Comment by BenjaminRuston** [comment]
pushed up my own suggestions,, will merge if passes CI.

please mark with `do not merge` if you see any issue with this @mer-a-o

**Comment by mer-a-o** [comment]
@BenjaminRuston Changes look good to me. We may need to update the other compo converter with `--time_range` later to be consistent.

**Pull request merged**
